rust/rope/src/lib.rs

```diff
@@ -1101,30 +1153,6 @@ impl<'a> PartialEq<Rope> for Cow<'a, str> {
 }
 
 #[test]
-fn concat_small() {
-    let a = Rope::from("hello ");
-    let b = Rope::from("world");
-    assert_eq!("hello world", String::from(a.clone() + b.clone()));
-    assert!("hello world" == a + b);
-}
-
-#[test]
-fn subrange_small() {
-    let a = Rope::from("hello world");
-    let b = a.slice(1, 9);
-    assert_eq!("ello wor", String::from(&b));
-    let c = b.slice(1, 7);
-    assert_eq!("llo wo", String::from(c));
-}
-
-#[test]
-fn replace_small() {
-    let mut a = Rope::from("hello world");
-    a.edit_str(1, 9, "era");
-    assert_eq!("herald", String::from(a));
-}
-
```

Comment:
I moved them to be doc tests (see above)

Josh Holmer <notifications@github.com> schrieb am Di., 3. Mai 2016 um 19:23:

> In rust/rope/src/lib.rs
> <https://github.com/google/xi-editor/pull/55#discussion_r61920328>:
>
> > -#[test]
> > -fn subrange_small() {
> > -    let a = Rope::from("hello world");
> > -    let b = a.slice(1, 9);
> > -    assert_eq!("ello wor", String::from(&b));
> > -    let c = b.slice(1, 7);
> > -    assert_eq!("llo wo", String::from(c));
> > -}
> > -
> > -#[test]
> > -fn replace_small() {
> > -    let mut a = Rope::from("hello world");
> > -    a.edit_str(1, 9, "era");
> > -    assert_eq!("herald", String::from(a));
> > -}
> > -
>
> Is there a reason for removing these tests?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/google/xi-editor/pull/55/files/abf3ee9fa08efa8579625d67940ac62963febcbf#r61920328>
>